contracts/Manager.sol

```diff
@@ -170,9 +170,9 @@ contract Manager is Ownable {
     }
 
     /// Set the seigniorage, in BPS.
-    function setSegniorage(uint256 _seigniorage) external onlyOwner {
-        seigniorage = _seigniorage;
+    function setSeigniorage(uint256 _seigniorage) external onlyOwner {
         emit SeigniorageChanged(seigniorage, _seigniorage);
+        seigniorage = _seigniorage;
```

Comment:
The ordering of the lines previously resulted in the event emitting the wrong information.